basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -785,6 +785,18 @@ public static void addGlobalData(Site site, Properties props, Properties custom,
 		M_log.debug("before custom="+custom);
 		LTI2Util.substituteCustom(custom, lti2subst);
 		M_log.debug("after custom="+custom);
+		
+		//TODO check if TII activated?
+		//substitute resources params
+		if(custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID) != null){
```

Comment:
Although you need the `if` for the debugging you can just call `setProperty()` as it doesn't change anything if the value is `null`.